src/lib/git/GitCommit.js

```diff
@@ -95,38 +145,45 @@ class GitCommit {
       return !c.change[0] || !c.change[1] || c.change[0].cid !== c.change[1].cid
     })
   }
+}
 
-  static fetchCommitAndParents(cid, total, onUpdate) {
-    let count = 0
-    const fetched = {}
-    let commits = []
-    const fetch = (currCid) => {
-      // If we've collected enough commits, we're done
-      if (count >= total) {
-        return
-      }
+class GitCommit {
+  constructor(data, cid) {
+    Object.assign(this, data)
+    this.cid = cid
 
-      // If we've already fetched this commit, skip it
-      if (fetched[currCid]) {
-        return
-      }
-      fetched[currCid] = true
+    this.author.moment = moment(this.author.date, 'X ZZ')
+    this.committer.moment  = moment(this.author.date, 'X ZZ')
+
+    this.summary = this.message
+    if (this.message.length > 80) {
```

Comment:
Might want to make it a constant at the top of the file so it's at least more visible. Something like `COMMIT_MSG_LEN_MAX`